### PR TITLE
OEL-2794: Update components to run on Drupal 10.2.x

### PR DESCRIPTION
**Pull request by AaronGilMartinez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,15 @@ The OpenEuropa Starter Content provides a series of content types to quickstart
 
 **Please note:** This project is not compatible with oe_content content types.
 
+#### Drupal 10.1
```
Comment (drishu) [inline README.md]: remove this as oe_content is a dev dependency

File: composer.json [hunk 2]
```diff
@@ -58,6 +58,18 @@
             "locations": {
                 "web-root": "./build"
             }
+        },
+        "patches": {
```
Comment (drishu) [inline composer.json]: do we really need this in our project?

**Review by drishu — CHANGES_REQUESTED** [review]
I think we can change to php 8.2 in docker for local development

**Review by drishu — CHANGES_REQUESTED** [review]
We can also safely drop version 9 from all core_version_requirement

File: README.md (hunk 1 above)
Comment (AaronGilMartinez) [reply README.md]: Removed

File: composer.json (hunk 2 above)
Comment (AaronGilMartinez) [reply composer.json]: Removed

**Review by drishu — APPROVED** [review]

**Review by brummbar — APPROVED** [review]

**Pull request merged**